modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/util/OpenAPIDeserializer.java

```diff
@@ -2711,7 +2715,11 @@ public ApiResponse getResponse(ObjectNode node, String location, ParseResult res
                 }else{
                     apiResponse.set$ref(ref.textValue());
                 }
-                 return apiResponse;
+                Map <String,Object> extensions = getExtensions(node);
+                if (extensions != null && extensions.size() > 0) {
+                  apiResponse.setExtensions(extensions);
+                }
+                return apiResponse;
```

Comment:
Needed for getting proto index extension when 204 (no content) is the only response. For 204, the proto index codegen hint will be provided at the response level extensions.

Reply:
actually this will be helpful in all response codes, whenever the response definition is reused in path definition